CMakeLists.txt

```diff
@@ -67,23 +67,30 @@ execute_process(
   OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 
+
 set(CLANG_CXXFLAGS "")
 set(CLANG_INCLUDEDIR "${LLVM_SRC}/tools/clang/include" "${LLVM_BUILD}/tools/clang/include")
+
+set(ALIVE_DIR "${CMAKE_SOURCE_DIR}/third_party/alive2")
+set(ALIVE_LDFLAGS "-L ${ALIVE_DIR}/build -lalive2")
+
 set(CLANG_LIBS "-lclangCodeGen -lclangTooling -lclangRewrite -lclangFrontend -lclangAnalysis -lclangParse -lclangSerialization -lclangSema -lclangEdit -lclangAnalysis -lclangAST -lclangDriver -lclangLex -lclangBasic")
 
 set(GTEST_CXXFLAGS "-DGTEST_HAS_RTTI=0")
 set(GTEST_INCLUDEDIR "${LLVM_SRC}/utils/unittest/googletest/include")
 set(GTEST_LIBS "-lgtest_main -lgtest")
 
-set(LLVM_CXXFLAGS "${LLVM_CXXFLAGS} -std=c++11")
+set(LLVM_CXXFLAGS "${LLVM_CXXFLAGS} -std=c++17")
```

Comment:
does this have any global implications that we need to think more carefully about?

Reply:
Everything seems to work so far and there is a good chance LLVM might switch within the next year.